src/quantum/eng/Generate-Client.ps1

```diff
@@ -0,0 +1,100 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Is this file a replacement for Generate-DataPlane-Client.ps1, or will we only use this to generate the control plane one?